.coveragerc

```diff
@@ -0,0 +1,4 @@
+[report]
+exclude_lines =
+    pragma: no cover
+    if TYPE_CHECKING:
```

Comment:
I noticed that `codecov` was counting `TYPE_CHECKING` blocks as uncovered, but they can be auto-ignored with this minimal configuration

Without this change, code coverage is still increased, so this change could be extracted for a separate PR

---

Reference:

- Advice from 2019 should still be relevant: https://github.com/nedbat/coveragepy/issues/831#issuecomment-517778185
- Pinned documentation from latest release: https://github.com/nedbat/coveragepy/blob/a5f37e0ba1d00329d4e8aa68e3d17478feca157c/doc/excluding.rst#L148
- Live documentation: https://coverage.readthedocs.io/en/7.2.7/config.html#config